docker/gitpod/osgeo-geo.Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM osgeo/gdal
```

Comment:
osgeo-geo is a bit of a odd name. Shouldn't that be osgeo-gdal, or perhaps ubuntu-full to better reflect its lineage ?

Reply:
That name is a total brain fart! I will change to osgeo-gdal, which is what I'd intended from the beginning.